uaclient/entitlements/anbox.py

```diff
@@ -0,0 +1,80 @@
+from typing import Tuple
+
+from uaclient import event_logger, messages, system
+from uaclient.entitlements.repo import RepoEntitlement
+from uaclient.files.state_files import (
+    AnboxCloudCredentials,
+    AnboxCloudData,
+    anbox_cloud_credentials_file,
+)
+from uaclient.types import MessagingOperationsDict, StaticAffordance
+
+event = event_logger.get_event_logger()
+
+
+class AnboxEntitlement(RepoEntitlement):
```

Comment:
I think we need something to disable `enableByDefault` since it is currently `true` and we may need to leave it as true for a while until `pro enable anbox-cloud` is the only way being used to enable anbox-cloud

Reply:
We may want an integration test for that too

Reply:
That's fair because of the resourceTokens. But maybe there is another way to do that. I will double check